data/json/vehicle_groups.json

```diff
@@ -229,5 +229,14 @@
             [ "handjack", 150 ],
             [ "wheelbarrow", 100 ]
         ]
+    },
+    {
+        "id": "school_vehicles",
+        "type": "vehicle_group",
+        "//": "School bus or fire truck found near school locations",
```

Comment:
That was not my idea - see https://github.com/CleverRaven/Cataclysm-DDA/blob/master/src/mapgen.cpp#L5829.